interactions/api/models/gw.py

```diff
@@ -251,13 +251,20 @@ class GuildMember(ClientSerializerMixin):
     roles: Optional[List[str]] = field(default=None)
     user: Optional[User] = field(converter=User, default=None, add_client=True)
     nick: Optional[str] = field(default=None)
-    avatar: Optional[str] = field(default=None)
+    _avatar: Optional[str] = field(default=None, discord_name="avatar")
     joined_at: Optional[datetime] = field(converter=datetime.fromisoformat, default=None)
     premium_since: Optional[datetime] = field(converter=datetime.fromisoformat, default=None)
     deaf: Optional[bool] = field(default=None)
     mute: Optional[bool] = field(default=None)
     pending: Optional[bool] = field(default=None)
 
+    def __str__(self) -> str:
+        return self.name or ""
+
+    @property
+    def avatar(self) -> Optional[str]:
```

Comment:
Add docstring

Reply:
The other method also doesn't have one xD

Reply:
Soooo do we add or no

Reply:
Ig no since it just returns the normal avatar